compile-errors ??? BUG?

The following error occurred in the `zephir build` implementation, I think it is Bug.
```shell
lll.c
array.c
backtrace.c
concat.c
debug.c
exception.c
exit.c
fcall.c
file.c
E:\Projects\lll\ext\kernel\file.c(155) : warning C4018: “<”: 有符号/无符号不匹配
E:\Projects\lll\ext\kernel\file.c(238) : warning C4018: “>”: 有符号/无符号不匹配
E:\Projects\lll\ext\kernel\file.c(248) : warning C4018: “<”: 有符号/无符号不匹配
filter.c
E:\Projects\lll\ext\kernel\filter.c(57) : warning C4018: “<”: 有符号/无符号不匹配
E:\Projects\lll\ext\kernel\filter.c(98) : warning C4018: “<”: 有符号/无符号不匹配
E:\Projects\lll\ext\kernel\filter.c(131) : warning C4018: “<”: 有符号/无符号不匹配
E:\Projects\lll\ext\kernel\filter.c(235) : warning C4018: “<”: 有符号/无符号不匹配
hash.c
iterator.c
main.c
memory.c
object.c
E:\Projects\lll\ext\kernel\object.c(115) : error C2275: “uint32_t”: 将此类型用作表达式非法
        D:\Server\PHP\php-5.6.4-devel-VC11-x86/include\win32/php_stdint.h(93) : 参见“uint32_t”的声明
E:\Projects\lll\ext\kernel\object.c(115) : error C2146: 语法错误: 缺少“;”(在标识符“i”的前面)
E:\Projects\lll\ext\kernel\object.c(115) : error C2065: “i”: 未声明的标识符
E:\Projects\lll\ext\kernel\object.c(117) : error C2065: “i”: 未声明的标识符
E:\Projects\lll\ext\kernel\object.c(117) : warning C4018: “<”: 有符号/无符号不匹配
E:\Projects\lll\ext\kernel\object.c(118) : error C2065: “i”: 未声明的标识符
```
D:\Server\zephir\ext\kernel\object.c

![2](https://cloud.githubusercontent.com/assets/1439155/6650567/3e72b9cc-ca54-11e4-8910-3caecd5c50a7.png)

D:\Server\PHP\php-5.6.4-devel-VC11-x86/include\win32/php_stdint.h(93) 

![3](https://cloud.githubusercontent.com/assets/1439155/6650571/5696bac6-ca54-11e4-971e-fed24438824a.png)

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/9545905-compile-errors-bug?utm_campaign=plugin&utm_content=tracker%2F280146&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F280146&utm_medium=issues&utm_source=github).
</bountysource-plugin>

--- a/ext/kernel/object.c
+++ b/ext/kernel/object.c
@@ -104,6 +104,7 @@ int zephir_is_instance_of(zval *object, const char *class_name, unsigned int cla
 int zephir_zval_is_traversable(zval *object TSRMLS_DC) {
 
 	zend_class_entry *ce;
+	unsigned int i;
 
 	if (Z_TYPE_P(object) == IS_OBJECT) {
 		ce = Z_OBJCE_P(object);
@@ -112,10 +113,11 @@ int zephir_zval_is_traversable(zval *object TSRMLS_DC) {
 			return 1;
 		}
 
-		uint32_t i;
-
 		for (i = 0; i < ce->num_interfaces; i++) {
-			if (ce->interfaces[i] == zend_ce_aggregate || ce->interfaces[i] == zend_ce_iterator || ce->interfaces[i] == zend_ce_traversable) {
+			if (ce->interfaces[i] == zend_ce_aggregate ||
+				ce->interfaces[i] == zend_ce_iterator ||
+				ce->interfaces[i] == zend_ce_traversable
+			) {
 				return 1;
 			}
 		}

